client/blocks/eligibility-warnings/index.jsx

```diff
@@ -54,7 +54,10 @@ export const EligibilityWarnings = ( {
 	return (
 		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
-
+			<TrackComponentView
+				eventName="calypso_automated_transfer_eligibility_show_warnings"
```

Comment:
The other eligibility events `calypso_automated_transfer_eligibility_eligible` and `calypso_automated_transfer_eligibility_ineligible` are fired when ever eligibility conditions are checked (https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js#L88). 

From my testing this event only fires when we show warnings to the user. Or specifically when ever `<EligibilityWarnings />` gets rendered.